include/swift/Option/Options.td

```diff
@@ -929,6 +929,12 @@ def Wwarning : Separate<["-"], "Wwarning">,
   MetaVarName<"<diagnostic_group>">,
   HelpText<"Treat this warning group as warning">;
 
+def Wsuppress : Separate<["-"], "Wsuppress">,
+  Group<warning_treating_Group>,
+  Flags<[FrontendOption, HelpHidden]>,
+  MetaVarName<"<diagnostic_group>">,
+  HelpText<"Suppress this warning group">;
+
```

Comment:
This was a big discussion in the review cycle of https://github.com/swiftlang/swift-evolution/blob/main/proposals/0443-warning-control-flags.md and it was intentionally not added to that proposal.

Reply:
Removed, thanks for the pointer. `DefaultIgnore` fits better here.